lib/layout/box_layout.dart

```diff
@@ -221,19 +222,26 @@ class BoxLayoutState extends WidgetState<BoxLayout> with TemplatedWidgetState {
     }
 
     Widget rtn = BoxWrapper(
-        ignoresMargin: true,  
         boxController: widget._controller,
-        widget: flutter.InkWell(
-            splashColor: flutter.Colors.transparent,
-            onTap: widget._controller.onTap == null ? null : () =>
-                ScreenController().executeAction(context, widget._controller.onTap!),
-            child: Padding(
-                padding: widget._controller.padding ?? const EdgeInsets.all(0),
-                child: boxWidget
-            )
-        )
+        widget: boxWidget,
+        ignoresMargin: true,
     );
 
+
+     if (widget._controller.onTap != null) {
+      rtn = flutter.InkWell(
```

Comment:
Don't just copy the code over from another class and ignore what's in this class. You are missing splashColor: transparent. We need to set this as we don't expose custom splashColor on this class.